src/bokeh/tile_providers.py

```diff
@@ -27,19 +27,19 @@
     Raises:
         ValueError, if the specified provider can not be found
 
-    Example:
+    **Example:**
```

Comment:
@mosc9575 is the emph necessary? AFAIK "Example:" is recognized by sphinx as a "known" section in google style docstrings. Are we certain adding emph does not interfere with that?

Reply:
Let me investigate a bit first, I'd like to actually build the docs locally and take a look and see if I can figure out what is going on. It just may be a couple of days before I can set aside time (thanks for patience as always!)

Reply:
In case you can experiment before me, here is the page of example google docstringsfor sphinx, maybe it is the order as you say (re-ordering would be fine with me in that case)

https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google